frontend/src/components/Card.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { FC } from "react";
+import styled, { css } from "styled-components";
+
+interface iCard {
+  priority: number;
```

Comment:
I see on line 16 that `priority` has a default value, which actually means it's optional.
(if there is no priority it set the value to one). So in your interface the value should be optional. Now it's required